ras_party/controllers/account_controller.py

```diff
@@ -235,12 +249,14 @@ def change_respondent_password(token, payload, tran, session):
     party_id = respondent.party_uuid
 
     try:
-        NotifyGateway(current_app.config).confirm_password_change(email_address, personalisation, str(party_id))
+        NotifyGateway(current_app.config).confirm_password_change(
+            email_address, personalisation, str(party_id))
     except RasNotifyError:
         log.error("Error sending notification email for party_id {}".format(party_id))
 
     # This ensures the log message is only written once the DB transaction is committed
-    tran.on_success(lambda: log.info('Respondent with id {} has changed their password'.format(party_id)))
+    tran.on_success(lambda: log.info(
+        'Respondent with id {} has changed their password'.format(party_id)))
```

Comment:
Not sure if party has been updated with them but sdc logging standards say we should log this as `log.info('Respondent has changed their password', party_id=party_id)`

Reply:
I agree this and it falls in line with the suggested logging standard https://github.com/ONSdigital/software-engineer-community/pull/6

Reply:
To update the logging within party is on the technical backlog

Reply:
@andrewmil - thanks for pointing this out. I'll leave this for now as it's on another card.

Reply:
I added a card to review all the repos logging standards to the tech debt board.